src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,114 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double SIZE_MULTIPLIER = 1.5;
+    private static final String WRONG_INDEX = "Index is out of list size";
+    private Object[] values;
+    private int size;
+    private int capacity;
+
+    {
+        values = new Object[DEFAULT_CAPACITY];
+        capacity = DEFAULT_CAPACITY;
+    }
```

Comment:
```suggestion
    public ArrayList() {
        values = (T[]) new Object[DEFAULT_CAPACITY];
        capacity = DEFAULT_CAPACITY;
    }
```
just use constructor